src/platforms/mesa/server/kms/display.cpp

```diff
@@ -488,6 +488,7 @@ void mgm::Display::configure_locked(
             // Each vector<KMSOutput> is a single GPU memory domain
             std::vector<std::vector<std::shared_ptr<KMSOutput>>> kms_output_groups;
             glm::mat2 transformation;
+            geom::Size physical_size{};
```

Comment:
I'm not entirely wild about the name `physical_size` here, because we *also* have a field for the actual physical of the display :grin: 

Maybe `resolution`? That's got a long history of meaning the correct thing.